vendor/engines/saml_authentication/app/controllers/saml_authentication/sessions_controller.rb

```diff
@@ -2,13 +2,69 @@ module SamlAuthentication
 
   class SessionsController < Devise::SamlSessionsController
 
+    # Some SSO providers are strict and require that the name_identifier matches
```

Comment:
I think this comment can go down by the method definition, to match with how you're doing it for `:store_winning_strategy`.